controllers/predictor_controller.go

```diff
@@ -257,6 +263,11 @@ func (pr *PredictorReconciler) handlePredictorNotFound(ctx context.Context,
 	_, err := mmc.DeleteVModel(deleteCtx, &mmeshapi.DeleteVModelRequest{VModelId: name.Name, Owner: sourceId})
 	if err != nil {
 		if isNoAddresses(err) {
+			// Work-around to prevent Non-MM InferenceService indefinite reconcile loop.
+			if sourceId == InferenceServiceCRSourceId {
+				pr.Log.Info("Ignoring InferenceService")
```

Comment:
I'm not sure we want to log this every time since there will be a lot of such messages in the case where there are no model-mesh pods and lots of other non-MM `InferenceService` activity.